docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,307 @@
+  --cluster-name <cloud-cluster-name> \
+  --context <cloud-cluster-context>
+
+# Accept from ITS
+clusteradm --context <its-context> accept --clusters <cloud-cluster-name>
+```
+
+## Troubleshooting
+
+### Common Issues and Solutions
+
+**1. CSR not appearing**
+```shell
+# Check if the join command completed successfully (look for Running pods)
+kubectl --context <wec-context> get pods -n open-cluster-management-agent
+
+# Verify network connectivity
+kubectl --context <wec-context> get events --all-namespaces | grep -i error
+```
+
+**2. CSR stuck in Pending state**
+```shell
+# Check CSR details
+kubectl --context <its-context> get csr <csr-name> -o yaml
+
+# Manually approve if needed
+kubectl --context <its-context> certificate approve <csr-name>
+```
+
+**3. ManagedCluster shows as Unknown**
+```shell
+# Check OCM agent logs
+kubectl --context <wec-context> logs -n open-cluster-management-agent deployment/klusterlet-registration-agent
+
+# Verify ITS connectivity
+kubectl --context <wec-context> get events -n open-cluster-management-agent
+```
+
+**4. Network connectivity issues**
+```shell
+# Test connectivity from WEC to ITS
+kubectl --context <wec-context> run test-connectivity --image=curlimages/curl --rm -it -- curl -k <its-api-server>
+
+# Check firewall rules and security groups
+```
+
+### Diagnostic Commands
+
+```shell
+# Check OCM components status
+kubectl --context <its-context> get pods -n open-cluster-management-hub
+kubectl --context <wec-context> get pods -n open-cluster-management-agent
+
+# Verify ManagedCluster resource
+kubectl --context <its-context> describe managedcluster <wec-name>
+
+# Check agent logs
+kubectl --context <wec-context> logs -n open-cluster-management-agent deployment/klusterlet-registration-agent
+kubectl --context <wec-context> logs -n open-cluster-management-agent deployment/klusterlet-work-agent
+```
+
+## Managing Registered WECs
+
+### Viewing WEC Status
+
+```shell
+# List all WECs
+kubectl --context <its-context> get managedclusters
+
+# Get detailed status
+kubectl --context <its-context> get managedcluster <wec-name> -o yaml
+
+# Check WEC labels
+kubectl --context <its-context> get managedcluster <wec-name> --show-labels
+```
+
+### Updating WEC Labels
+
+```shell
+# Add new labels
+kubectl --context <its-context> label managedcluster <wec-name> new-label=value
+
+# Remove labels
+kubectl --context <its-context> label managedcluster <wec-name> old-label-
+
+# Update existing labels
+kubectl --context <its-context> label managedcluster <wec-name> existing-label=new-value --overwrite
+```
+
+### Deregistering a WEC
+
+To remove a WEC from KubeStellar:
+
+```shell
+# Delete the ManagedCluster resource
+kubectl --context <its-context> delete managedcluster <wec-name>
+
+# Clean up the WEC (optional)
+kubectl --context <wec-context> delete namespace open-cluster-management-agent
+kubectl --context <wec-context> delete namespace open-cluster-management-agent-addon
+```
+
+## Next Steps
+
```

Comment:
Shouldn't there be a pointer to the general big picture (https://rishi-jat.github.io/kubestellar/doc-wec-registration/direct/user-guide-intro/#the-full-story) somewhere in here?